serde-generate/src/python3.rs

```diff
@@ -252,7 +268,7 @@ import typing
         writeln!(self.out, "\nclass {}:", name)?;
         self.out.indent();
         self.output_comment(&name)?;
-        if self.generator.config.serialization {
+        if self.generator.config.serialization && !self.has_custom_code(&name) {
```

Comment:
why this guard? when has custom code, we lost serialization?

Reply:
We output `pass` in the else branch.

Reply:
or, you think we should just pull out serialization into custom code?

Reply:
Oh my bad it should be a OR.